### PR TITLE
Shim autoclass command

### PR DESCRIPTION
**Pull request by NickGoog**

### CODE REVIEW

File: gslib/commands/autoclass.py [hunk 1]
```diff
@@ -64,6 +65,12 @@
 _set_help_text = CreateHelpText(_SET_SYNOPSIS, _SET_DESCRIPTION)
 _get_help_text = CreateHelpText(_GET_SYNOPSIS, _GET_DESCRIPTION)
 
+GCLOUD_FORMAT_STRING = ('--format=value[separator=\"\"](name.sub("^", "gs://")'
```
Comment (dilipped) [inline gslib/commands/autoclass.py:68]: It looks like this for me

```
$ gcloud alpha storage buckets list gs://gsutil_test_bucket-4 --format='value[separator=\"\"](name.sub("^", "gs://").sub("$", ":\n  Enabled: "),autoclass.enabled.yesno(True, False).sub("$", "\n  Toggle Time: "),autoclass.toggleTime.yesno(no=None))'
gs://gsutil_test_bucket-4:
  Enabled: ""False
  Toggle Time: ""None
```

Maybe try 
```
$ gcloud alpha storage buckets list gs://gsutil_test_bucket-4 --format="value[separator='\n'](format('gs://{}:', name), format('  Enabled: {}', autoclass.enabled.yesno(True, False)), format('  Toggle Time: {}', autoclass.toggleTime))"
gs://gsutil_test_bucket-4:
  Enabled: False
  Toggle Time: None
```

Comment (NickGoog) [reply gslib/commands/autoclass.py:68]: Without change:
```
$ CLOUDSDK_CORE_PASS_CREDENTIALS_TO_GSUTIL=True ./gsutil autoclass get gs://b1
gs://b1:
  Enabled: True
  Toggle Time: 2022-11-02T19:18:11.202000+00:00

$ CLOUDSDK_CORE_PASS_CREDENTIALS_TO_GSUTIL=True ./gsutil autoclass get gs://b2
gs://b2:
  Enabled: False
  Toggle Time: None
```
With change:
```
$ CLOUDSDK_CORE_PASS_CREDENTIALS_TO_GSUTIL=True ./gsutil autoclass get gs://b1
gs://b1:
  Enabled: True
  Toggle Time: 2022-11-02T19:18:11.202000+00:00
$ CLOUDSDK_CORE_PASS_CREDENTIALS_TO_GSUTIL=True ./gsutil autoclass get gs://b2
gs://b2:
  Enabled: False
  Toggle Time: None
```

I'll go with yours b/c it's more portable outside the shim

Comment (dilipped) [reply gslib/commands/autoclass.py]: Ah interesting, so it was a copy-paste issue I believe?

**Review by dilipped — APPROVED** [review]

File: gslib/commands/autoclass.py (hunk 1 above)
Comment (NickGoog) [reply gslib/commands/autoclass.py]: Maybe related to the terminal handling of quotations

**Pull request merged**